.travis.yml

```diff
@@ -63,7 +63,7 @@ before_install:
   - shellcheck $(find . -type f -name "*.sh")
   - shellcheck bin/*
   # check Dockerfile
-  - docker run --rm -i lukasmartinelli/hadolint < Dockerfile
+  - docker run --rm -i lukasmartinelli/hadolint hadolint --ignore DL3008 --ignore DL3015 - < Dockerfile
```

Comment:
* We want pinned versions in apt (https://github.com/lukasmartinelli/hadolint/wiki/DL3008)
* It's a good idea to not install reccommends (--no-install-recommends)

Please put this checks back and fix the code accordingly.